spec/src/main/asciidoc/release_notes.asciidoc

```diff
@@ -0,0 +1,42 @@
+//
+// Copyright (c) 2016-2017 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// You may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// Contributors:
+// John D. Ament
+
+[[release_notes_11]]
+== Release Notes for MicroProfile Config 1.1
+
+The following changes occurred in the 1.1 release, compared to 1.0
+
+A full list of changes may be found on the link:https://github.com/eclipse/microprofile-config/milestone/2?closed=1[MicroProfile Config 1.1 Milestone]
+
+=== API Changes
+
+The API for MicroProfile Config has not changed with 1.1.
```

Comment:
Do we need to list the individual api at all? The following paragraph is sufficient.

Reply:
I guess we don't, if there's no breaking change, but why not if it's already written?

Reply:
The individual APIs are not listed, so I'm not sure what you mean.  I do suspect that these three sections should be listed within each release.

Reply:
This sentence above said no change in MP Config. In the following sentence said there was a change in ConfigSource. I did not know whether the sentence _"The API for MicroProfile Config has not changed with 1.1."_ meant there was no change in Config.java or you meant Config specification. It cannot be later as there was a change in ConfigSource.java. My suggestion is to remove the sentence completely.

Reply:
I guess John meant that the public API wasn't change, only SPI has some changes. I would just remove the above sentence, because the sentence below explains that the only change is in the `ConfigSource` and that's enough.

Reply:
So you want to remove the entire section?

Reply:
@Emily-Jiang I don't see how you're reading that.  "MicroProfile Config" is the proper name for this component.

Reply:
@johnament I was trying to find some reason for link this sentence in question with the following sentence about what change goes in. Shall we just remove the following sentence ("The API for MicroProfile Config has not changed with 1.1.") and everything will be clear.